Doc/library/uuid.rst

```diff
@@ -186,7 +186,7 @@ The :mod:`uuid` module defines the following functions:
 .. function:: uuid3(namespace, name)
 
    Generate a UUID based on the MD5 hash of a namespace identifier (which is a
-   UUID) and a name (which is a string).
+   UUID) and a name (which is a byte sequence or a UTF-8-encoded string).
```

Comment:
```suggestion
   UUID) and a name (which is a byte sequence or a string).
```
Strings aren't encoded in any encoding, they are sequences of characters.

Also, I'd prefer to be precise and say "bytes object" (linking to :class:`bytes`) instead of the vague term "byte sequence".

Reply:
It'd be good to be explicit that the string would then be UTF-8 encoded, if provided.

Reply:
I changed "bytes sequence" to "bytes object" (with link). For the second part, I removed the word "encoded" and left "UTF-8", is it a valid compromise?

Reply:
"UTF-8 string" is still not a meaningful statement. Strings in Python aren't UTF-8 or ASCII or any other specific encoding; they are a sequence of abstract characters. We can say something like "a string which will be encoded using UTF-8".

Reply:
Thank you for the explaination. I've updated the lines with your suggestion.